salvo/src/lib/docker_volume.py

```diff
@@ -55,8 +55,11 @@ def generate_volume_config(output_dir, test_dir=''):
   volume_json = {}
   try:
     volume_json = json.loads(MessageToJson(volume_cfg))
-  except Error as serialize_error:
-    log.exception(f"Could not build volume json object: {serialize_error}")
+  except json.decoder.JSONDecodeError as decode_error:
+    log.error(f"Could not build volume json object: {decode_error}")
+    raise
```

Comment:
(pre-existing) If a function directly raises an exception, we should document it in the "Raises:" section of its docstring.

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#24-exceptions

Reply:
Updated